src/localization/locales/en-US.json

```diff
@@ -646,6 +646,13 @@
         "table_search_filter_results_other": "{{rows.length}} matches found for “{{query}}“",
         "table_search_filter_results_one": "{{rows.length}} match found for “{{query}}“",
         "table_search_filter_clear": "Clear search",
+        "table_pagination_label": "Table pagination",
+        "table_pagination_previous_page": "Go to previous page",
+        "table_pagination_next_page": "Go to next page",
+        "table_pagination_displayed": "{{from}} - {{to}} of {{count}}",
+        "table_sort_desc": "Sort by DESC",
+        "table_sort_asc": "Sort by ASC",
```

Comment:
```suggestion
        "table_sort_desc": "Sort descinding",
        "table_sort_asc": "Sort ascending",
```

Reply:
@paulschreiber Is that a typo for descending?

Reply:
Yes. fixed.